testing.py

```diff
@@ -25,9 +25,11 @@ def _get_variables_header_fields() -> str:
     if labels_len != len(variables):
         sys.stderr.write("The number of defined labels doesn't match the number of defined variables")
         sys.stderr.write("  continuing processing")
+        sys.stderr.write("\n")
```

Comment:
Why this change? Should it be a general change in the template repo?

Reply:
I added that change because it is hard to read when neither labels_len or units_len is equal to len(variables) since both print statements are written on the same line. I changed it so that the \n is at the end of each line so that there does not need to be a separate sys.stderr.write for each block